CHANGELOG.md

```diff
@@ -30,6 +30,13 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
+## 0.33.1
+
+**November 23, 2023**
+
+- Deprecation announcement for geometry, acquired_before, acquired_after and custom_filter parameters in `get_asset` to enforce use of `pystac_client` search in the future.
```

Comment:
I checked the changelog: we don't "announce deprecations", we only add these entries in 2 cases:
- When a functionality is marked as deprecated.
- When a deprecated functionality is removed.

Reply:
I added the deprecation now, as it is a removed feature.